### PR TITLE
Revised Databricks UDF README

### PR DESCRIPTION
**Pull request by manny-silva**

### CODE REVIEW

File: tokenize_from_csv/README.md [hunk 1]
```diff
@@ -1,58 +1,73 @@
...
 
-The config must contain:
+## Set up the UDF
 
-- **creds_path**: path to the credentials.json for the service account you want to connect to the vault
-- **vault_url_prefix**: the API URL up to ".com". Ex: https://abcdefg.vault.skyflowapis.com
-- **account_id**: the Skyflow accountID associated with the vault
-- **vault_id**: the ID for your vault
+1. Upload the *credentials.json* file to a Databricks volumne. This example assumes the path is `/Volumes/main/default/test_volume/credentials.json`.
+1. Create a JSON object with you Skyflow information as follows:
```
Comment (josandt3) [inline tokenize_from_csv/README.md]: with your

File: tokenize_from_csv/README.md [hunk 2]
```diff
@@ -1,58 +1,73 @@
...
+
+1. Create a map that maps the CSV columns to the Skyflow columns. For example, if `fake_data.csv` has the `customer_names`, `user_email_address`, and `social_security_number` columns, the map could be:
+
+    | CSV column | Skyflow column |
+    |------------|----------------|
+    | customer_names | name |
+    | user_email_address | email_address |
+    | social_security_number | ssn |
+
+1. Copy the contents of the `tokenize_from_csv.py` file into a Databricks notebook. Click the `run` button on top of the notebook. This registers (or re-registers) the UDF.
```
Comment (josandt3) [inline tokenize_from_csv/README.md]: 1,2,3

**Review by josandt3 — APPROVED** [review]
Minor fixes, thank you for the improvement

File: tokenize_from_csv/README.md [hunk 3]
```diff
@@ -1,58 +1,73 @@
...
+This workbook is a sample Databricks UDF that reads data from a CSV file, inserts it into a Skyflow vault, and stores the associated token to Databricks. You can use it as a reference to build your own UDFs.
 
-##### Prerequisites
+## Before you start
 
-Create a Skyflow vault with a schema that is compatible with the CSV input to be tokenized.
+Before you start, you need the following:
 
-This workflow requires the following inputs to be passed when calling the UDF:
+* A Databricks volumne you can write to.
```
Comment (joicejohnskyflow) [inline tokenize_from_csv/README.md]: Volume

File: tokenize_from_csv/README.md [hunk 4]
```diff
@@ -1,58 +1,73 @@
...
+  * The path to the CSV file in your Databricks workspace. This example uses the path is `/Volumes/main/default/test_volume/fake_data.csv`.
 
-The config must contain:
+## Set up the UDF
 
-- **creds_path**: path to the credentials.json for the service account you want to connect to the vault
-- **vault_url_prefix**: the API URL up to ".com". Ex: https://abcdefg.vault.skyflowapis.com
-- **account_id**: the Skyflow accountID associated with the vault
-- **vault_id**: the ID for your vault
+1. Upload the *credentials.json* file to a Databricks volumne. This example assumes the path is `/Volumes/main/default/test_volume/credentials.json`.
```
Comment (joicejohnskyflow) [inline tokenize_from_csv/README.md]: Volume

**Comment by joicejohnskyflow** [comment]
Minor nits, Looks good apart from that

File: tokenize_from_csv/README.md (hunk 3 above)
Comment (hawkeyexl) [reply tokenize_from_csv/README.md]: Fixed

File: tokenize_from_csv/README.md (hunk 4 above)
Comment (hawkeyexl) [reply tokenize_from_csv/README.md]: Fixed. Why do I do this?

File: tokenize_from_csv/README.md (hunk 1 above)
Comment (hawkeyexl) [reply tokenize_from_csv/README.md]: Fixed

File: tokenize_from_csv/README.md (hunk 2 above)
Comment (hawkeyexl) [reply tokenize_from_csv/README.md]: Keeping it as-is. Markdown autoenumerates the list items. Keeping it all `1`s keeps the source clean, and you don't need to worry about renumbering every time you update the number or order of steps.

Comment (josandt3) [reply tokenize_from_csv/README.md]: Super useful, thank you

**Pull request merged**